src/types/entity.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2023  David Kellner
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {AliasSetT, AliasT} from './aliases';
+import {IdentifierSetT} from './identifiers';
+import {LazyLoaded} from './utils';
+
+
+export type EntityTypeString =
+	| 'Author'
+	| 'Edition'
+	| 'EditionGroup'
+	| 'Publisher'
+	| 'Series'
+	| 'Work';
```

Comment:
This type is already defined elsewhere, but I think this would be the better place for this definition.
If you agree, I will delete the older definition and point all import statements to the new one.
https://github.com/metabrainz/bookbrainz-data-js/blob/8af33f049a292e6632938cd6c53e1b2a81d9af43/src/func/types.ts#L25-L26